guides/common/attributes-satellite.adoc

```diff
@@ -50,6 +50,8 @@
 :client-os: {RHEL}
 :client-os-context: red-hat-enterprise-linux
 :Cockpit: RHEL web console
+:CockpitL: the RHEL web console
+:CockpitA: The RHEL web console
```

Comment:
```suggestion
:the-Cockpit: the RHEL web console
:The-Cockpit: The RHEL web console
```
An existing attribute that uses an article follows the syntax `:article-term:` so I suggest to stick to the hyphenated syntax for consistency: 
https://github.com/theforeman/foreman-documentation/blob/c01debb13cfe42decae242f18f1ba952d9cc7022/guides/common/attributes-satellite.adoc#L83

I also think `L` and `A` might be too cryptic and not immediately clear... so how about `the-` and `The-`? It's longer but clear at first sight. But I'd be curious if anyone else has a better idea.

Reply:
As we discussed in person, the syntax you suggested probably won't work. Attributes in Asciidoc do not seem to be case-sensitive, and I had to change my former attempt (`CockpitA` and `Cockpita`) to something different by the second commit in this PR.

Reply:
How about:

- `:the-cockpit:`
- `:thec-cockpit:` for "capitalized"